fluster/decoders/ffmpeg.py

```diff
@@ -492,3 +492,24 @@ class FFmpegAV1VCudaDecoder(FFmpegCudaDecoder):
     """FFmpeg CUDA decoder for AV1"""
 
     codec = Codec.AV1
+
+
+class FFmpegQsvDecoder(FFmpegDecoder):
+    """Generic class for FFmpeg QSV (Intel Quick Sync Video) decoder"""
+
+    hw_acceleration = True
+    api = "QSV"
+    init_hw_device = "qsv=hw"
```

Comment:
This argument is not obligatory, it works with and without it.
It also works with modification `init_hw_device = "qsv"`